app/models/results/sql_generator.rb

```diff
@@ -23,8 +23,7 @@ def generate
       select("questions.qtype_name AS question_type").
       select("users.name AS submitter_name").
       select("answers.id AS answer_id").
-      select("answers.inst_num AS instance_num").
-      select("answers.rank AS rank").
+      select("answers.old_rank AS rank").
```

Comment:
Let's remove this line altogether for now. I don't think it's necessary and it complicates things big time.